src/test/java/teammates/test/cases/util/RecaptchaVerifierTest.java

```diff
@@ -0,0 +1,23 @@
+package teammates.test.cases.util;
+
+import org.testng.annotations.Test;
+
+import teammates.common.util.RecaptchaVerifier;
+import teammates.test.cases.BaseTestCase;
+
+/**
+ * SUT: {@link RecaptchaVerifier}.
+ */
+public class RecaptchaVerifierTest extends BaseTestCase {
+
+    @Test
+    public void testIsVerificationSuccessful() {
+        ______TS("null or empty response");
+        assertFalse(RecaptchaVerifier.isVerificationSuccessful(null));
+        assertFalse(RecaptchaVerifier.isVerificationSuccessful(""));
+
+        ______TS("any other key"); // true if using the official secret test key
+        assertTrue(RecaptchaVerifier.isVerificationSuccessful("testResponseKey"));
```

Comment:
sorry i missed this out. Personally i m against writing unit tests for service that is dependent on third party servers. Hence for this test, pls write a surrogate for the third party server to mimic its behavior e.g. just sending these strings and get back the json responses for different scenarios.

Reply:
Ok, I'll extract lines 62-75 of `RecaptchaVerifier` into a private method and test that.

But, I don't think I can test `RecaptchaVerifier#isVerificationSuccessful` any further (in terms of exceptions and verification failures), given the current implementation.

Reply:
>The other way could be to wrap the API request in a new non-utility class (and not test that class). This way, we can introduce a stub. But, I don't know whether this is worth doing...

what about just a subclass of the RecaptchaVerifier class? I hope it wont take too many lines?